src/bin/tikv-ctl.rs

```diff
@@ -927,7 +958,25 @@ fn main() {
                         .help("PD endpoints"),
                 ),
         )
+        .subcommand(SubCommand::with_name("bad-regions").about("get all regions with corrupt raft"))
         .subcommand(
+            SubCommand::with_name("metrics")
+                .about("print the metrics")
+                .arg(
+                Arg::with_name("tag")
+                    .short("t")
+                    .long("tag")
+                    .takes_value(true)
+                    .multiple(true)
+                    .use_delimiter(true)
+                    .require_delimiter(true)
+                    .value_delimiter(",")
+                    .default_value(METRICS_PROMETHEUS)
```

Comment:
Maybe use an empty value as default?

Reply:
We set `METRICS_PROMETHEUS ` as default since  we need to dump Prometheus only in most case